pkg/services/featuremgmt/toggles_gen.json

```
@@ -596,6 +596,22 @@
         "codeowner": "@grafana/grafana-frontend-platform"
       }
     },
+    {
+      "metadata": {
+        "name": "assetSriChecks",
+        "resourceVersion": "1739984409734",
+        "creationTimestamp": "2025-02-19T15:56:59Z",
+        "annotations": {
+          "grafana.app/updatedTimestamp": "2025-02-19 17:00:09.734088 +0000 UTC"
+        }
+      },
+      "spec": {
+        "description": "Enables SRI checks for Grafana JavaScript assets",
+        "stage": "experimental",
+        "codeowner": "@grafana/plugins-platform-backend",
```

Maybe this should be `@grafana/plugins-platform-frontend`?
```suggestion
        "codeowner": "@grafana/plugins-platform-frontend",
```

---

Ah well spotted this should probably be `@grafana/frontend-ops`. 👍